assets/js/src/form_editor/store/controls.jsx

```diff
@@ -167,6 +167,15 @@ export default {
     dispatch('core/block-editor').resetBlocks(updatedBlocks);
   },
 
+  TUTORIAL_DISMISS() {
```

Comment:
We have a common control `CALL_API` so there is no need to create a new control for that. `CALL_API` also returns info if an action was a success or failed so you can react on that in your actions generator function.

Here is an [example usage in the form editor](https://github.com/mailpoet/mailpoet/blob/master/assets/js/src/form_editor/store/actions.ts#L207-L220).

I think it would be nicer to reuse this control so that we don't repeat ajax calls code and error handling etc.